src/wp-admin/plugins.php

```diff
@@ -495,7 +495,7 @@
 
 				// Return early if all selected plugins already have auto-updates enabled or disabled.
 				// Must use non-strict comparison, so that array order is not treated as significant.
-				if ( $new_auto_updates == $auto_updates ) { // phpcs:ignore WordPress.PHP.StrictComparisons.LooseComparison
+				if ( $new_auto_updates === $auto_updates ) { // phpcs:ignore WordPress.PHP.StrictComparisons.LooseComparison
```

Comment:
While this change will probably not be accepted - I'd suggest for future patches to remove the `phpcs:ignore` comment if an issue is fixed and should no longer be ignored ;-)